CHANGES.md

```diff
@@ -31,6 +31,10 @@ To be released.
     constructor.  [[#666], [#917]]
  -  Added `BigInteger`-typed `previousTotalDifficulty` parameter to
     `Block<T>.Mine()` static method.  [[#666], [#917]]
+ -  `Pong()` constructor has changed:  [[459], [#919]]
```

Comment:
I guess this mention may not be needed because `Pong` isn't exposed to Libplanet users.

Reply:
+1, except message types should remain as they are part of the network protocol.